cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/RunCommand.java

```diff
@@ -83,6 +84,8 @@ public class RunCommand implements BLauncherCmd {
     @CommandLine.Option(names = "--debug", hidden = true)
     private String debugPort;
 
+    @CommandLine.Option(names = "--watch", description = "listen for file changes and automatically re-run the project")
```

Comment:
```suggestion
    @CommandLine.Option(names = "--watch", description = "listen for file changes and automatically rerun the project")
```

Reply:
Changed listen to watch. According to chatgpt,

> The correct term is "re-run" when used as a verb, meaning to run again. So, the phrase should be:
> 
> "watch for file changes and automatically re-run the project"